lib/src/controls/better_player_controls_configuration.dart

```diff
@@ -76,7 +77,10 @@ class BetterPlayerControlsConfiguration {
   final Duration controlsHideTime;
 
   ///Custom controls, it will override Material/Cupertino controls
-  final Widget customControls;
+  final Widget Function(BetterPlayerController controller) customControls;
```

Comment:
final Widget Function(BetterPlayerController) customControlsBuilder seems to be better name for me.